tests/gem5/gpu/README.md

```diff
@@ -6,3 +6,18 @@ To run these tests by themselves, you can run the following command in the tests
 ```bash
 ./main.py run gem5/gem5-resources --length=very-long
 ```
+
+To obtain the input graphs needed to run the Pannotia tests, run the following
+in tests/gpu-pannotia-resources/pannotia-datasets:
+
+```bash
+gsutil -m cp -r \
+"gs://dist.gem5.org/dist/develop/datasets/pannotia/README.txt" \
+"gs://dist.gem5.org/dist/develop/datasets/pannotia/bc" \
+"gs://dist.gem5.org/dist/develop/datasets/pannotia/color" \
+"gs://dist.gem5.org/dist/develop/datasets/pannotia/floydwarshall" \
+"gs://dist.gem5.org/dist/develop/datasets/pannotia/mis" \
+"gs://dist.gem5.org/dist/develop/datasets/pannotia/pagerank" \
+"gs://dist.gem5.org/dist/develop/datasets/pannotia/sssp" \
+.
+```
```

Comment:
We shouldnt be having the user do this. Just have the python script wget the resources was needed.

Let's do this the cheap way: put the wget's (or whatever the python equivalent is) in tests_gpu_pannotia.py. Just have a little part at the top of the file that checks if tests/gem5/resources/pannotia exits, and, if not, creates the directory downloads all the binaries and datafiles into it.